docs/book/advanced_guide/guide-aws-gcp-azure.md

```diff
@@ -0,0 +1,246 @@
+---
+description: Deploy pipelines to the public cloud.
+---
+
+# Guide for cloud-specific deployments
+
+This guide will show how you can run the pipeline in Kubeflow Pipelines deployed
+to a public cloud cluster. We will start with some pre-requisites and then move
+on to show the integration of your cloud provider's components with your ZenML
+stack. In addition to configuring the stack components in this guide, you can
+optionally run a pipeline step on a specialized cloud backend too. Check out the
+step operators [guide](step-operators.md) for that!
+
+## Pre-requisites
+
+### Orchestrator
+
+{% tabs %}
+{% tab title="AWS" %}
+
+* Have an existing
+  AWS [EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html)
+  set up.
+* Download and [install](https://kubernetes.io/docs/tasks/tools/) `kubectl`
```

Comment:
@schustmi not sure I understood your point here.

Reply:
I think what I meant here: We essentially have a list of prerequisites or steps that you need to follow to get it running. But installing the AWS cli (and same for azure and gcp) is mentioned in sort of a half sentence "make sure you have this as well" instead of a separate point "Install the AWS CLI [like this](http://howto-install-aws.cli)"